### PR TITLE
Update K3s hardening guide to CIS Kubernetes Benchmark 1.6

### PR DESCRIPTION
**Pull request by macedogm**
Update K3s hardening guide to CIS Kubernetes Benchmark 1.6. The CIS profile used for the update was [k3s-cis-1.6-hardened](https://github.com/rancher/security-scan/tree/master/package/cfg/k3s-cis-1.6-hardened).

The pages that were updated are:
- [K3s > CIS Hardening Guide](https://rancher.com/docs/k3s/latest/en/security/hardening_guide/)
- [K3s > CIS Self Assessment Guide](https://rancher.com/docs/k3s/latest/en/security/self_assessment/)

Signed-off-by: Guilherme Macedo <guilherme.macedo@suse.com>

### CODE REVIEW

**Comment by macedogm** [comment]
FYI @samjustus

**Review by brandond — APPROVED** [review]

File: scripts/converters/headers/header-k3s.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: CIS Self Assessment Guide
+weight: 90
+---
+
+### CIS Kubernetes Benchmark v1.6 - K3s with Kubernetes v1.17 to v1.21
+
+#### Overview
+
+This document is a companion to the K3s security hardening guide. The hardening guide provides prescriptive guidance for hardening a production installation of K3s, and this benchmark guide is meant to help you evaluate the level of security of the hardened cluster against each control in the CIS Kubernetes Benchmark. It is to be used by K3s operators, security teams, auditors, and decision-makers.
```
Comment (jtravee) [inline scripts/converters/headers/header-k3s.md:10]:
```suggestion
This document is a companion to the [K3s security hardening guide]({{<baseurl>}}/k3s/latest/en/security/hardening_guide/). The hardening guide provides prescriptive guidance for hardening a production installation of K3s, and this benchmark guide is meant to help you evaluate the level of security of the hardened cluster against each control in the CIS Kubernetes Benchmark. It is to be used by K3s operators, security teams, auditors, and decision-makers.
```

File: scripts/converters/headers/header-2.6.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+weight: 101
+---
+
+### CIS v1.6 Kubernetes Benchmark - Rancher v2.6 with Kubernetes v1.18 to v1.21
+
+[Click here to download a PDF version of this document](https://releases.rancher.com/documents/security/2.6/Rancher_v2-6_CIS_v1-6_Benchmark_Assessment.pdf).
+
+#### Overview
+
+This document is a companion to the Rancher v2.6 security hardening guide. The hardening guide provides prescriptive guidance for hardening a production installation of Rancher, and this benchmark guide is meant to help you evaluate the level of security of the hardened cluster against each control in the benchmark.
```
Comment (jtravee) [inline scripts/converters/headers/header-2.6.md:12]:
```suggestion
This document is a companion to the [Rancher v2.6 security hardening guide]({{<baseurl>}}/rancher/v2.6/en/security/hardening-guides/). The hardening guide provides prescriptive guidance for hardening a production installation of Rancher, and this benchmark guide is meant to help you evaluate the level of security of the hardened cluster against each control in the benchmark.
```

**Comment by macedogm** [comment]
@jtravee thanks for the suggestions!

**Pull request merged**